### PR TITLE
AC-394 - Improve UI for PatientPhotoActivity

### PR DESCRIPTION
**Pull request by adhi1419**

### CODE REVIEW

File: openmrs-client/build.gradle [hunk 1]
```diff
@@ -19,8 +19,8 @@ apply plugin: 'me.tatarka.retrolambda'
 def version = "2.6.1-beta"
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.2"
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
```
Comment (AvijitGhosh82) [inline openmrs-client/build.gradle]: Don't change this. The CI fails.

**Comment by shivtej1505** [comment]
@adhi1419 , I would say the better way will be get the Uri of the image and then show it using [Intent.ACTION_VIEW](https://developer.android.com/reference/android/content/Intent.html#ACTION_VIEW) .
In that case, we won't need to write a new style and we can use the feature provided by Android System.

**Comment by dkayiwa** [comment]
@adhi1419 are you still working on this?

**Comment by dkayiwa** [comment]
@adhi1419 ping me to reopen when you resume working on this.

**Pull request closed without merging**